website/docs/r/gcp_secret_impersonated_account.html.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: "vault"
+page_title: "Vault: vault_gcp_secret_impersonated_account resource"
+sidebar_current: "docs-vault-resource-gcp-secret-impersonated-account"
+description: |-
+  Creates a Impersonated Account for the GCP Secret Backend for Vault.
+---
+
+# vault\_gcp\_secret\_impersonated\_account
+
+Creates a Impersonated Account in the [GCP Secrets Engine](https://www.vaultproject.io/docs/secrets/gcp/index.html) for Vault.
+
+Each [impersonated account](https://www.vaultproject.io/docs/secrets/gcp/index.html#impersonated-accounts) is tied to a separately managed
+Service Account.
+
+## Example Usage
+
+```hcl
+resource "google_service_account" "this" {
+  account_id = "my-awesome-account"
+}
+
+resource "vault_gcp_secret_backend" "gcp" {
+  path        = "gcp"
+  credentials = "${file("credentials.json")}"
+}
+
+resource "vault_gcp_secret_impersonated_account" "impersonated_account" {
+  backend        = vault_gcp_secret_backend.gcp.path
+
+  impersonated_account  = "this"
+  service_account_email = google_service_account.this.email
+  token_scopes          = ["https://www.googleapis.com/auth/cloud-platform"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Required, Forces new resource) Path where the GCP Secrets Engine is mounted
+
+* `impersonated_account` - (Required, Forces new resource) Name of the Impersonated Account to create
+
+* `service_account_email` - (Required, Forces new resource) Email of the GCP service account to manage.
```

Comment:
Nit/Question: Would saying "Email of the GCP service account to impersonate" be more descriptive? This is more of a question on my part as I'm not a consistent user of GCP, so maybe "manage" is the standard term here and is actually better.

Reply:
Switching from "manage" to "impersonate" makes sense to me.